### DISCUSSION TITLE
All seems to be fine thus far i do especially like how simple going the process was, but security default issue

### DISCUSSION THREAD

**Discussion by genaro1098273** [tteck/Proxmox#3228] [Q&A] [answered]
Everything seems fine so far. I especially like how simple and easy the process is. However, there's a security issue: when you first start up the containers, there are no default firewall settings blocking incoming traffic. I understand that some websites need to be open to the world wide web, but I suggest adding a prominent notice at the top of the page, along with the supported versions in red. This way, people will immediately know that when they create the container, there's an open door to the world. I configured it using advanced setting more than one container, and even when i used proxmox firewall it wasn't blocking anything, i had to manually install a firewall and put secure settings....seems like something should have been common sense, but anyways yeah. thanks

**Comment by tteck** [comment]
Nothing is open to the "world wide web" unless you make it that way. 

> seems like something should have been common sense, but anyways yeah. thanks

😆 All that I can say is, learn.

**Comment by genaro1098273** [comment]
Explain?
How is that so; i need ro know because i installed a firewall and if theres no need than it can cause issues
________________________________

**Reply by tteck** [reply, marked as answer]
I think you might be confused about the difference between incoming requests from your local network and those from the internet.

**Comment by genaro1098273** [comment]
Also; can you explain why it disabled the functionality of the proxmox  firewall server pertaining to that lxc, yes i rebooted and even reloaded the pve-firewall after install
________________________________

**Reply by tteck** [reply]
Read and understand https://pve.proxmox.com/wiki/Firewall

Did you enable the firewall for the container?

![Screenshot 2024-06-18 2 08 26 AM](https://github.com/tteck/Proxmox/assets/21342670/ffe63baa-9a58-436e-864b-d0e02816f6f6)

**Reply by genaro1098273** [reply]
Yeah, it was disabled after the script installation, but when I usually create a container manually, it's enabled by default. Thanks.

**Reply by genaro1098273** [reply]
worked

**Comment by genaro1098273** [comment]
When comparing the simplicity of the scripts, it's just standard. Thanks for writing these scripts for free. The issue was a miscalculation on my part. I am aware of cluster firewalls and PVE firewalls and how they function. The new issue is People use things like Cloudflare tunnels and such, so basic firewall settings are no longer efficient in such use case scenarios. Even though Cloudflare protects you from external threats, you still have local ones to worry about, but I can't complain
________________________________
